app/components/elements/forms/checkbox_component.rb

```diff
@@ -4,6 +4,12 @@ module Elements
   module Forms
     # Component for form checkbox field
     class CheckboxComponent < FieldComponent
```

Comment:
Shouldn't whether it is checked be driven by the form?

Reply:
Done, see below